### PR TITLE
MUIC-618: [Android] - Extract colors for header title and home button

### PR DESCRIPTION
**Pull request by deniss-mc**
# https://glia.atlassian.net/browse/MUIC-618

### CODE REVIEW

**Review by dukhovnyi — APPROVED** [review]
for me, it looks OK, but please wait for @AndriiHorishniiMOC  review

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java [hunk 1]
```diff
@@ -58,32 +58,69 @@ public AppBarView(@NonNull Context context, @Nullable AttributeSet attrs, int de
                 Utils.getTypedArrayIntegerValue(typedArray,
                         context,
                         R.styleable.AppBarView_android_backgroundTint,
-                        R.attr.gliaBrandPrimaryColor));
-        int lightColor = ContextCompat.getColor(
+                        R.attr.gliaBrandPrimaryColor
+                )
+        );
+
+        int titleTintColor = ContextCompat.getColor(
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:65]: this method is getting very long. can it be broken down? like `setTitleTextColor()`

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java [hunk 2]
```diff
@@ -58,32 +58,69 @@ public AppBarView(@NonNull Context context, @Nullable AttributeSet attrs, int de
...
+
+    private void setTypeface() {
+        if (this.theme.getFontRes() != null) {
+            Typeface fontFamily = ResourcesCompat.getFont(
+                    this.getContext(),
+                    this.theme.getFontRes()
+            );
+            titleView.setTypeface(fontFamily);
+        }
+
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:117]: unnecessary empty line

**Review by jakob-mc — CHANGES_REQUESTED** [review]

**Comment by dukhovnyi** [comment]
Andrii is busy, ok to merge without his review when it will be ready.

File: widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java (hunk 2 above)
Comment (deniss-mc) [reply widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java:117]: Please validate in MUIC-619

**Pull request closed without merging**